### PR TITLE
Make llvm pass build feature dependent

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

File: fuzzers/fuzzbench/Cargo.toml [hunk 1]
```diff
@@ -29,7 +29,7 @@ libafl = { path = "../../libafl/" }
 libafl_bolts = { path = "../../libafl_bolts/" }
 libafl_targets = { path = "../../libafl_targets/", features = ["sancov_pcguard_hitcounts", "sancov_cmplog", "libfuzzer"] }
 # TODO Include it only when building cc
-libafl_cc = { path = "../../libafl_cc/" }
+libafl_cc = { path = "../../libafl_cc/", features = ["autotokens", "cmplog-routines"] }
```
Comment (domenukk) [inline fuzzers/fuzzbench/Cargo.toml:32]: Let's make those two `default`

**Comment by domenukk** [comment]
Also should the names start with `llvm_`?
In theory we could support other compilers

**Comment by tokatoka** [comment]
> Also should the names start with llvm_?
In theory we could support other compilers

No other part of libafl_cc already assumes the compiler is clang. it won't work with gcc even if we added it.
if we do this we better have different folder in libafl-gcc

**Comment by tokatoka** [comment]
build.rs is for clang..
but why does anybody want to use clang when you certainly can install clang

**Comment by domenukk** [comment]
Some targets only work with gcc ¯\_(ツ)_/¯

**Pull request merged**